doc/X-OR_tutorial/X-OR_tutorial.rst

```diff
@@ -834,7 +834,7 @@ The new algorithm we are going to use to recover the :math:`x_i` from :math:`\su
 
```

Comment:
"but it is too tightly linked to the computation order" : I didn't get what's the "computation order" here. Is it the order of the computations ?